chrome/content/test/local_storage.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @fileOverview Test suite for 
+ * chrome/content/privly-applications/shared/javascripts/local_storage.js
```

Comment:
"This tests using the preferences object to store strings. The more general test cases can be found..."